src/routes/AppRouter.tsx

```diff
@@ -32,6 +33,7 @@ const AppRouter: React.FC = () => {
         <Route path="/terms-of-service" element={<TermsOfService />} />
         <Route path="/contact-us" element={<ContactUs />} />
         <Route path="/blog" element={<Blog />} />
+        <Route path="/blog/article-name" element={<Article />} />
```

Comment:
change "article-name" to be a route parameters, path should be "/blog/:article-name"

Reply:
done